### DISCUSSION TITLE
I was approved the student benefits but NO E-mail sent to me.

### DISCUSSION THREAD

**Discussion by JasonStephen** [community/community#55602] [GitHub Education] [answered]
### Hi everyone,

I was approved the student benefits but NO E-mail sent to me.
Approved Infromation: Congrats! Your request for @JasonStephen was approved and your benefits will be available soon.
Another Infromation: Hi, ***! You were last verified as a student on yesterday. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I wait all day without any reply.
I would like to know how long I need to wait after "approved". Will I receive an email from Github? Will it be sent to my main email address, my backup email address or still my application email address？
Repeat: I ask a question if it has been Approved rather than rejected。

**Comment by jge162** [comment, marked as answer]
You will get an email to the registered email yes. Wait is from 24-72 hours when I was approved

**Reply by JasonStephen** [reply]
Thank you. I will wait patiently. If i have any other questions, I will consult you again

**Reply by ganireddikumar** [reply]
Does it got approved ??

**Reply by ganireddikumar** [reply]
Got Mail? Same problem here

**Reply by JasonStephen** [reply]
> Does it got approved ??

Now no. If over 72 hours not get E-mail, I will find another help.

**Reply by MiskaWasTaken** [reply]
same problem here too

**Reply by exababy** [reply]
same

**Reply by JasonStephen** [reply]
> > > Does it got approved ??
> > 
> > Now no. If over 72 hours not get E-mail, I will find another help.
> 
> Did you get the email now? I have the same problem as you

Yes I got the E-mail

**Reply by JasonStephen** [reply]
> Got Mail? Same problem here

I got the email from my verified email

**Reply by javierdev776** [reply]
> > Got Mail? Same problem here
> 
> I got the email from my verified email

Really how long does it took for verified mail man

**Reply by jge162** [reply]
It’s 24-72 hours

**Reply by JasonStephen** [reply]
> It’s 24-72 hours

Yes, I got it and applied for Copilot, Thanks

**Reply by JasonStephen** [reply]
> > > Got Mail? Same problem here
> > 
> > 
> > I got the email from my verified email
> 
> Really how long does it took for verified mail man

24 to 72 hours, mine was 72 hours

**Reply by swjs49** [reply]
no, far more than 72 hours, and still no email received
.

**Reply by swjs49** [reply]
f**k

**Reply by tintinthedev** [reply]
I think almost 72 hours passed and I still got nothing

**Comment by gojohnny69** [comment]
its more than 2 days but not approved man same problem

**Reply by Tobear2** [reply]
Help please

**Comment by Tobear2** [comment]
Please help I am having trouble myself.

**Reply by zhengxiaoran-996** [reply]
Have you been approved

**Comment by forrccc** [comment]
more than 3 days for me, same problem

**Reply by swjs49** [reply]
the same

**Comment by CoolGenius-123** [comment]
Have you used any student email or just verified using documents?

**Comment by forrccc** [comment]
> Have you used any student email or just verified using documents?

I verified using documents sir

**Reply by CoolGenius-123** [reply]
Can you tell what documents you have used ?

**Reply by forrccc** [reply]
I took a photo of my school portal as it was an option in it... Also the website shows me this message:
**Approved Submitted Jul 1, 2023
Congrats! Your request for [@adityasinghi3761](https://github.com/adityasinghi3761) was approved and your benefits will be available soon.**

**Comment by samus-aran** [comment]
Hi @JasonStephen 👋 

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.

**Comment by Tobear2** [comment]
Wtf

**Comment by QXuanYun** [comment]
Same problem.... I was approved yesterday...Thought I WILL get the package 'SOON'

**Reply by zhengxiaoran-996** [reply]
Have you been approved

**Comment by reda332211** [comment]
Same problem

**Reply by zhengxiaoran-996** [reply]
Have you been approved

**Comment by ibrbou** [comment]
Same problem for me too

**Reply by zhengxiaoran-996** [reply]
I encountered a similar issue to yours

**Reply by ibrbou** [reply]
> I encountered a similar issue to yours

You will be approved in the coming days, i've received an email this morning, it took me around 3 days i guess.

**Comment by arunkumar-da** [comment]
Congrats! Your request for [@arunkumar-da](https://github.com/arunkumar-da) was approved and your benefits will be available soon. but i didn't get the benefit still yet please anyone respond to me

**Reply by xyz020801** [reply]
i got the same problem as you,it just say "will be available soon"

**Comment by ayush222006** [comment]
I am facing this same problem @samus-aran 

EDIT= I got the beneifts in 4 days anyone seeing this please be assured your benefits are on the way.

**Reply by thanfot** [reply]
me too

**Reply by ghost** [reply]
me too

**Reply by ayush222006** [reply]
I got mine in 4 days @FallingAKS

**Comment by ghost** [comment]
really creepy&nbsp;


?
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月26日(星期一) 晚上8:12
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] I was approved the student benefits but NO E-mail sent to me. (Discussion #55602)





 
f**k

**Comment by qiang-xu123** [comment]
same ploblem,I have witing for 4 days
